lang/cs/Org.Apache.REEF.Driver/Bridge/ClrHandlerHelper.cs

```diff
@@ -87,21 +125,31 @@ public static ISet<string> GetCommandLineArguments()
                 }
                 catch (InjectionException e)
                 {
-                    string error = "Cannot inject command line arguments from driver bridge configuration. ";
+                    const string error = "Cannot inject command line arguments from driver bridge configuration.";
                     Exceptions.CaughtAndThrow(e, Level.Error, error, LOGGER);
-                    throw e;
+                    throw;
                 }
                 return arguments.Arguments;
             }
         }
 
+        /// <summary>
+        /// Allows additional Java classes to be included into the classpath by the user.
+        /// Generates a file named <see cref="Constants.GlobalUserSuppliedJavaLibraries"/> in the 
+        /// executing directory containing classpath information.
+        /// </summary>
+        /// <param name="classPaths">classpaths to the Java classes</param>
         public static void SupplyAdditionalClassPath(params string[] classPaths)
```

Comment:
I don't think this is used in the "new" client code paths. Shall we deprecate it?